tests/acceptance/21_methods/action_ifelapsed_respected.cf.sub

```diff
@@ -0,0 +1,23 @@
+bundle agent main
+{
+  methods:
+    "test"
+      action => expireafter_ten_sec;
+}
+
+body action expireafter_ten_sec
+{
+  expireafter => "10";
```

Comment:
Finally you might want to make this test a "timed' test, since it's going to sleep a long time. But I believe @kacf has already implemented `ifelapsed` and `expireafter` tests. Did you find them?